SmsSender.Test/IntegrationTest1.cs

```diff
@@ -0,0 +1,43 @@
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Xunit;
+
+namespace SmsSender
+{
+    public class IntegrationTest1
+        : IClassFixture<WebApplicationFactory<Startup>>
+    {
+        private readonly WebApplicationFactory<Startup> _factory;
+
+        public IntegrationTest1(WebApplicationFactory<Startup> factory)
+        {
+            _factory = factory;
+        }
+
+        [Theory]
+        [InlineData("/hello/anonymous", HttpStatusCode.OK, "text/plain; charset=utf-8")]
```

Comment:
@andresmoschini FYI 
The original code is ` [InlineData("/WeatherForecast", HttpStatusCode.OK, "application/json; charset=utf-8")]`
I change that because I removed `WeatherForecast` controller from this project.